analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -184,16 +184,18 @@ open class Npm(
         val npmrcFile = getUserHomeDirectory().resolve(".npmrc")
         if (npmrcFile.isFile) {
             npmrcFile.forEachLine { line ->
-                val (key, value) = line.split('=', limit = 2).map { it.trim() }
-
-                val proxyUrl = value.takeIf { it.matches(HTTP_REGEX) } ?: when (key) {
-                    "proxy" -> "http://$value"
-                    "https-proxy" -> "https://$value"
-                    else -> ""
-                }
+                val keyAndValue = line.split('=', limit = 2).map { it.trim() }
+                if (keyAndValue.size == 2) {
+                    val (key, value) = keyAndValue
+                    val proxyUrl = value.takeIf { it.matches(HTTP_REGEX) } ?: when (key) {
+                        "proxy" -> "http://$value"
+                        "https-proxy" -> "https://$value"
+                        else -> ""
+                    }
 
-                if (proxyUrl.isNotEmpty()) {
-                    applyProxySettingsFromUrl(URL(proxyUrl))
+                    if (proxyUrl.isNotEmpty()) {
+                        applyProxySettingsFromUrl(URL(proxyUrl))
```

Comment:
out of scope of this change / PR: do we want to gracefully handle also `MalformedURLException`s?

Reply:
Good point, probably yes. I'll create a follow-up PR.

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/1682.